modules/dynamic-operator/api/v1alpha1/kuberlogicservice_types.go

```diff
@@ -111,11 +112,7 @@ func (in *KuberLogicService) TLSEnabled() bool {
 }
 
 func (in *KuberLogicService) GetHost() string {
-	var host string
```

Comment:
I think we need both scenarios with subdomains and custom domains

Reply:
Subdomains still exists, I just moved this logic to `modules/dynamic-apiserver/pkg/util/util.go` because there is no way to determine if domain was provided by user from here